test/test_main.cpp

```diff
@@ -65,6 +65,7 @@ int main(int argc, char** argv) {
   }
 
   manifold::PolygonParams().intermediateChecks = true;
+
```

Comment:
I would expect this to set to true and stay that way, and it seems to half-work, but then switch back only deep inside `polygon.cpp` when it's actually needed.